projects/canopy/src/lib/forms/input/input.stories.ts

```diff
@@ -27,6 +27,7 @@ interface KnobsConfig {
   hint?: string | null;
   icon?: string;
   iconButton?: boolean;
+  inputType?: 'number' | 'text' | 'tel'; // TODO add rest or make a string
```

Comment:
I would say add the remaining input types from the HTML spec rather than make a string, as it'll give us auto-completion and prevent typos.